homeassistant/components/group/fan.py

```diff
@@ -78,6 +79,20 @@ async def async_setup_platform(
     )
 
 
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Initialize Light Switch config entry."""
+    registry = er.async_get(hass)
+    entity_id = er.async_validate_entity_ids(
+        registry, config_entry.options[CONF_ENTITIES]
+    )
```

Comment:
This will fail setup if one of the group members is unknown by UUID, maybe we should not do that?